### ISSUE TITLE
underfitting for nvGesture?

### ISSUE THREAD

**Issue by asdfqwer2015** [ahmetgunduz/Real-time-GesRec#3]
Hi, I tried to train the classifier for nvgesture from scratch(the hyperparams come from run_online.sh, batchsize=8, resnext101, cls=25, lr=0.01, duration=32). But I found it's almost not fitting, after tens of epoch, the acc for trainset and valset is about 5%.
And I tried to increase batchsize to 16, the acc can however converged to 9%, it's still very low. I also tried to set norm_value=255 to normalize the input data to small range or smaller lr, but it didn't work.
Did I miss something?
BTW, the detector trained from scratch is well with acc about 80%.

**Comment by ahmetgunduz** [comment]
Hi, run_online.sh should only be used for real-time evaluation of not training.When you try to train, your strategy for specifying target label should b chosen properly because 99 percent of the batches will have frames from none and a gesture together. Most probably that is the reason you get a low accuracy. You should train your models first offline and then if you want to evaluate online you can use run_online.sh.

**Comment by ahmetgunduz** [comment]
Hi, run_online.sh is for evaluating offline trained models in real time basis. When you try to train online you will get low accuracy because frames  for each sample are sliding through video  and target labels(true labels) should be specified properly. You should first train your models using run_offline.sh then if you want to evaluate them in a real-time scenario, you can use run_online.sh. Hope this is clear.

**Comment by asdfqwer2015** [comment]
Thanks for your reply. I'll check the training samples.
:)
